virtcontainers/utils/utils.go

```diff
@@ -5,13 +5,28 @@
 
 package utils
 
+/*
+#include <linux/vhost.h>
```

Comment:
Linux-specific file so this probably should be moved to `virtcontainers/utils/utils_linux.go`.

See #379.

/cc @raravena80.

Reply:
sure  :smile: